app/models/picture_set.rb

```diff
@@ -85,13 +66,59 @@ def capture_job(num, date, dir, angle)
     def convert_thread(num, date, dir, angle)
       caption = OPTS.image_caption || date
       t = Thread.new do
-        Syscall.execute("time convert -caption '#{caption}' #{date}_#{num}.jpg -sample 600 -bordercolor Snow " \
-        "-density 100 -gravity center -pointsize #{OPTS.image_fontsize} " \
-        "-polaroid -#{angle} #{date}_#{num}#{POLAROID_SUFFIX}", dir: dir)
+        Syscall.execute("time convert -caption '#{caption}' #{date}_#{num}.jpg " \
+                                     '-sample 600 ' \
+                                     '-bordercolor Snow ' \
+                                     '-density 100' \
+                                     '-gravity center' \
+                                     "-pointsize #{OPTS.image_fontsize} " \
+                                     "-polaroid -#{angle} " \
+                                     "#{date}_#{num}#{POLAROID_SUFFIX}", dir: dir)
       end
       t.abort_on_exception = true
       t
     end
+
+  end
+
+  def initialize(date: nil)
+    @date = date
+    @path = "picture_sets/#{date}"
+    @dir = "#{PICTURE_PATH}/#{date}"
+    @animation = "#{date}#{ANIMATION_SUFFIX}"
+    @combined = "#{date}#{COMBINED_SUFFIX}"
+    @pictures = (1..4).map { |i| { polaroid: "#{date}_#{i}#{POLAROID_SUFFIX}", full: "#{date}_#{i}.jpg" } }
+  end
+
+  def destroy
+    Syscall.execute("rm -r #{date}", dir: PICTURE_PATH)
+  end
+
+  # Merge all polaroid previews to an animated gif
+  def create_animation(overwrite: false)
+    if File.exist?(File.join(dir, animation)) && !overwrite
+      Rails.logger.info "Skipping for existing animation #{dir}"
+    else
+      Rails.logger.info "Creating animation for #{dir}"
+      Syscall.execute("time convert -delay 60 #{date}_*#{POLAROID_SUFFIX} #{animation}", dir: dir)
+    end
+  end
+
+  # Create collage of all images
+  def combine_images(overwrite: false)
+    if File.exist?(File.join(dir, combined)) && !overwrite
```

Comment:
Maybe you can also introduce a flag for ``create_combine_images`` which we can set in the options. Then we don't need to comment it on the raspberry ...

Reply:
-> https://github.com/digitaltom/photobooth/issues/19